engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -660,6 +729,37 @@ public JSONObject getPorts(@PathParam("operatorName") String operatorName)
   }
 
   @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES + "/{moduleName}/ports")
```

Comment:
Updated the code to remove /moduleName/ports and /moduleName/ports/portname and reused url defined for operator to return module port information. Doing the testing for these changes.